metadataChecker.py

```diff
@@ -0,0 +1,70 @@
+import logging
+import os
+import eyed3
+
+VALID_FILE_TYPES = ["mp3"]
+
+#change log level for eyed3 module
+logging.getLogger('eyed3.id3.tag').setLevel(logging.ERROR)
+
+def check(absolute_path):
+    metadata_violation_counter = 0
+
+    for root, _, files in os.walk(absolute_path):
+        relative_path = '.' + root[len(absolute_path):]
+        metadata_violation_counter += _check_metadata(relative_path, root, files)
+
+    logging.info(f'count metadata violations: {metadata_violation_counter}')
+
+def _get_data_from_file_path(relative_file_path):
```

Comment:
a comment to the slicing magic maybe? i am confused 😵

Reply:
I guess you mean the slicing in 20-24?

The idea was to split the string by the os.sperator (eg '/').
We then access the elements from the back, because we don't know how many "elements" the file path contains

(eg `/a/b/c/d/artist/album/track`) -> `['a', 'b', 'c', 'd', 'artist', 'album', 'track']`

And the last slices just split the album into album name + release year and the track into track number and track name